server/src/resolvers.js

```diff
@@ -3,13 +3,23 @@ const {Mutation} = require('./resolvers/mutation');
 const {User} = require('./resolvers/user');
 const {Event} = require('./resolvers/event');
 const {EventBookClub} = require('./resolvers/eventBookClub');
-
+const {Tag} = require('./resolvers/tag');
 module.exports = {
   Query,
   Mutation,
   User,
   Event,
   EventBookClub,
-  Tag: {},
-  EventConnection: {},
+  Tag,
+  TagConnection: {
+    tags: async (parents, _, {dataSources}) =>{
+      return parents.tagNames;
```

Comment:
Is this only contains the name of tags? Modify this to return the whole tag object or just tag id.